### PR TITLE
sm.ms

### PR DESCRIPTION
**Pull request by SKEIDs**
#103016

### CODE REVIEW

File: ChineseFilter/sections/specific.txt [hunk 1]
```diff
@@ -938,6 +938,7 @@ olevod.com##.L4A
 olevod.com##.A4R
 olevod.com##.ad1
 olevod.com##.L1
+sm.ms##.wwads_abp
```
Comment (Sergey-Lyapin) [inline ChineseFilter/sections/specific.txt:941]: This is an antiadblock message, please move the rule to annoyances > antiadblock.txt

File: ChineseFilter/sections/specific.txt [hunk 2]
```diff
@@ -1049,4 +1050,4 @@ ifvod.tv##vg-player.ng-star-inserted > vg-pause-ads + div.caption.show > .inner
 ||coolpc.com.tw/home/img/right2.html
 ! https://github.com/AdguardTeam/AdguardFilters/issues/40396
 !+ PLATFORM(ios, ext_android_cb, ext_safari)
-m.xigua110.com##.ui-detail-info > div[style*="line-height:35px; text-align:center;"]
```
Comment (Sergey-Lyapin) [inline ChineseFilter/sections/specific.txt:1052]: please remove this empty line

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: ChineseFilter/sections/specific.txt [hunk 3]
```diff
@@ -1049,4 +1050,4 @@ ifvod.tv##vg-player.ng-star-inserted > vg-pause-ads + div.caption.show > .inner
...
\ No newline at end of file
+m.xigua110.com##.ui-detail-info > div[style*="line-height:35px; text-align:center;"]
```
Comment (Sergey-Lyapin) [inline ChineseFilter/sections/specific.txt:1053]: please remove the empty line

Comment (SKEIDs) [reply ChineseFilter/sections/specific.txt:1052]: done

File: ChineseFilter/sections/specific.txt (hunk 1 above)
Comment (SKEIDs) [reply ChineseFilter/sections/specific.txt]: ok. finished.

**Pull request merged**